### DISCUSSION TITLE
BT-53

### DISCUSSION THREAD

**Discussion by lepo** [OP-TED/eForms-SDK#706] [Q&A] [answered]
Hi,

Where can i find the definition of the Options Field BT-53? There is BT-54 with the description of the option, but i can't find BT-53 in fields.json neither in any notice definition.

Best Günter

**Comment by AnnaKSZ** [comment, marked as answer]
Hi,

Until the answer comes, I recommend the "Pointless Business Terms" page to your attention:
https://docs.ted.europa.eu/eforms/latest/schema/identifiers.html#pointlessDueToDesignSection

Regards
Anna
